site/styles/_nav.scss

```diff
@@ -307,19 +307,25 @@
     transform: translateX(-50%);
     width: 350px;
   }
+
   // Header Search Form Control
   .form-control-search {
     font-size: 10px;
-    color: #fff;
-    border-color: rgba(255,255,255,0.3);
-    background-color: rgba(255,255,255,0.3);
+    //border-color: rgba($color: #868686, $alpha: 0);
+    border-color: transparent;
+    background: rgba($color: $var-text, $alpha: 0.4);
+    
+    //background-color: color.invert(rgba($var-text, 0.5), $weight: 100%);
     @include media-breakpoint-up(md){
       font-size: 14px;
     }
     &::placeholder {
       text-align: left;
       color: #fff;
     }
+    &:hover {
+      border-color: #ced4da;
```

Comment:
Does this need to be themed?

Reply:
It is set to the same colour as the search results search bar. It works for both light and dark theme.

Reply:
Still probably shouldn't be a hard-coded colour... and I can't see `#ced4da` used anywhere else in core-template... maybe it should just use one of the existing colour variables?

Reply:
Changed to `border-color: rgba($body-color, 0.2);`